pkg/config/config.go

```diff
@@ -95,13 +94,96 @@ func (c *ConfigTx) Updated() *cb.Config {
 	return c.updated
 }
 
+// ComputeUpdate computes the ConfigUpdate from a base and modified config transaction.
+func (c *ConfigTx) ComputeUpdate(channelID string) (*cb.ConfigUpdate, error) {
+	if channelID == "" {
+		return nil, errors.New("channel ID is required")
+	}
+
+	updt, err := computeConfigUpdate(c.base, c.updated)
+	if err != nil {
+		return nil, fmt.Errorf("failed to compute update: %v", err)
+	}
+
+	updt.ChannelId = channelID
+
+	return updt, nil
+}
+
+// ChannelConfiguration returns a channel configuration value from a config transaction.
+func (c *ConfigTx) ChannelConfiguration() (Channel, error) {
+	channelGroup := c.base.ChannelGroup
+	var err error
+
+	_, ok := channelGroup.Values[ConsortiumKey]
+	var consortium string
+	if ok {
+		consortiumProto := &cb.Consortium{}
+		err := unmarshalConfigValueAtKey(channelGroup, ConsortiumKey, consortiumProto)
+		if err != nil {
+			return Channel{}, err
+		}
+		consortium = consortiumProto.Name
+	}
+
+	_, ok = channelGroup.Groups[ApplicationGroupKey]
```

Comment:
You can just inline the if check since you're not assigning the value for future use:
```
if _, ok := channelGroup.Groups[ApplicationGroupKey]; ok {...}
```

Reply:
done.